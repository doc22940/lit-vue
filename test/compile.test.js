const compile = require('../lib/compile')

test('simple', async () => {
  const result = await compile(
    `
  import { html } from 'lit-vue'

  const template: string = html\`
  <div></div>
  <style scoped>
  div {
    color: red;
  }
  </style>

  <custom-block name="i18n">
  hello
  </custom-block>
  \`

  export default {
    template
  }
  `,
    'foo.ts'
  )

  expect(result).toMatchSnapshot()
})
