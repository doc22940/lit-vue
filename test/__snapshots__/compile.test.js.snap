// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`simple 1`] = `
"
  <template>
  
  <div></div>
  

  
  
  </template>

  <script lang=\\"ts\\">
  const template: string = undefined;
export default {
  template
};
  </script>

  <style scoped>
  div {
    color: red;
  }
  </style>
  <i18n>
  hello
  </i18n>
    "
`;
